[devtools-translate] Portuguese - What's New in DevTools (Chrome 108) #4129 (#4161)
GoogleChrome/developer.chrome.com: site/pt/blog/new-in-devtools-108/index.md

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d6c1fea1e79b8373ff913a6d9919d097d1141254 #}

Issue no Chromium: [1178508](https://crbug.com/1178508)

<!-- ## Auto-detect XPath and text selectors in the Recorder panel {: #recorder } -->
## Detectação automática de XPath e seletores de texto no painel Gravador {: #recorder }

<!-- The **Recorder** panel now supports XPath and text selectors. [Start recording a user flow](/docs/devtools/recorder/#record) and the recorder automatically picks the XPath and shortest unique text of an element as selector if available. -->

O painel **Gravador** agora é compatível com XPath e seletores de texto. [Comece a gravar um fluxo de usuário](/docs/devtools/recorder/#record) e o gravador seleciona automaticamente o XPath e o texto único mais curto de um elemento como seletor, se disponível.

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/NJVIK95TtKaXxzNVoGI6.png", alt="XPath e seletores de texto no painel Gravador.", width="800", height="579" %}

{# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/7441acfff5d9dfd373742797d2db46a809c9df67 #}

> Issue no Chromiums: [1327206](https://crbug.com/1327206),[1327209] (https://crbug.com/1327209)

Reviewer: ```suggestion
Issue no Chromium: [1327206](https://crbug.com/1327206),[1327209] (https://crbug.com/1327209)
```

Author: Done

```

<!-- Transpilers and minifiers may turn them into comma-separated expressions. -->
Transpiladores e minificadores podem transformá-los em expressões separadas por vírgulas.

```js
function bar(){return foo(),foo(),42}
```  

<!-- This creates confusion during debugging because the stepping behavior is different between minified and authored code. It is even more confusing when using sourcemaps to debug the minified code in terms of the original code, as the developer is then looking at semicolons (which were under the hood turned into commas by the toolchain) but the debugger doesn't stop on them. -->

> Isso cria confusão durante a depuração porque o comportamento de depuração é diferente entre o código minificado e o de original. É ainda mais confuso ao usar mapas de origem para depurar o código minificado em termos do código original, pois o desenvolvedor está olhando para os pontos e vírgulas (que estavam sob o capô transformados em vírgulas pela cadeia de ferramentas), mas o depurador não parava neles .

Reviewer: ```suggestion
Isso cria confusão durante a depuração porque o comportamento de depuração é diferente entre o código minificado e o de original. É ainda mais confuso ao usar mapas de origem para depurar o código minificado em termos do código original, pois o desenvolvedor está olhando para os pontos e vírgulas (que estavam, por baixo dos panos, transformados em vírgulas pela cadeia de ferramentas), mas o depurador não parava neles.
```

Author: Done

{# https://chromium.googlesource.com/v8/v8/+/ade6d191c8566e3fe7331d2ef37e43760c7cb363 #}

Issue no Chromium: [1370200](https://crbug.com/1370200)

<!-- ## Improved Ignore list setting {: #ignore-list } -->
## Configuração de lista de ignorados aprimorada {: #ignore-list }
<!-- Go to **Settings** > **Ignore List**. DevTools improves the design to help you configure the rules to [ignore a single script or pattern of scripts](/docs/devtools/javascript/reference/#settings-ignore-list). -->

> Vá para **Configurações** > **Ignorar lista**. O DevTools melhora o design para ajudá-lo a configurar as regras para [ignorar um único script ou padrão de scripts](/docs/devtools/javascript/reference/#settings-ignore-list).

Reviewer: ```suggestion
Vá para **Configurações** > **Ignorar lista**. O DevTools melhora o design para te ajudar a configurar as regras para [ignorar um único script ou padrão de scripts](/docs/devtools/javascript/reference/#settings-ignore-list).
```

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/qazPkaZ3TkSrIBU89Jtn.png", alt="A guia Lista de Ignorados.", width="800", height="535" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/9441d8775b38b47db91bb5182f6349f3036d3751 #}

Issue no Chromium: [1356517](https://crbug.com/1356517)

<!-- ## Miscellaneous highlights {: #misc } -->
## Destaques diversos {: #misc }
<!-- These are some noteworthy fixes in this release: -->
Estas são algumas correções dignas de nota nesta versão:
<!-- - Autocomplete CSS property name in the **Styles** pane on pressing space. ([1343316](https://crbug.com/1343316)) -->
<!-- - Remove auto scroll in the **Element** panel’s breadcrumb. ([1369734](https://crbug.com/1369734)) -->

>   - Preencher automaticamente o nome da propriedade CSS no painel **Estilos** ao pressionar espaço. ([1343316](https://crbug.com/1343316))

Reviewer: ```suggestion
  - Preenchimento automático do nome da propriedade CSS no painel **Estilos** ao pressionar espaço. ([1343316](https://crbug.com/1343316))
```

Author: Done

>   - Remova a rolagem automática na trilha de navegação do painel **Element**. ([1369734](https://crbug.com/1369734))

Reviewer: ```suggestion
  - Remoção da rolagem automática na trilha de navegação do painel **Element**. ([1369734](https://crbug.com/1369734))
```

Author: Done
